src/api/api.js

```diff
@@ -1,15 +1,43 @@
-const BASE_URL = 'https://bootcamp-api.codeit.kr/api/sample/';
+const BASE_URL = 'https://bootcamp-api.codeit.kr/api/';
 
 const getSampleUserData = async () => {
-  const response = await fetch(`${BASE_URL}user`);
+  const response = await fetch(`${BASE_URL}sample/user`);
   const data = await response.json();
   return data;
 };
 
 const getSampleFolderData = async () => {
-  const response = await fetch(`${BASE_URL}folder`);
+  const response = await fetch(`${BASE_URL}sample/folder`);
   const { folder } = await response.json();
   return folder;
 };
 
-export { getSampleUserData, getSampleFolderData };
+const getTargetUserData = async userId => {
+  const response = await fetch(`${BASE_URL}users/${userId}`);
+  const { data } = await response.json();
+  return data[0];
+};
+
+const getTargetUserFolderListData = async userId => {
+  const response = await fetch(`${BASE_URL}users/${userId}/folders`);
+  const { data } = await response.json();
+  return data;
+};
+
+const getTargetUserFolderLinkListData = async (userId, folderId) => {
+  let response;
+  if (folderId !== 'all') {
+    response = await fetch(`${BASE_URL}users/${userId}/links?folderId=${folderId}`);
+  } else {
+    response = await fetch(`${BASE_URL}users/${userId}/links`);
+  }
+  const { data } = await response.json();
+  return data;
```

Comment:
let, if/else문 없이 이렇게도 바꿔볼 수 있겠네요.

```suggestion
  const api = `${BASE_URL}users/${userId}/links${forderId === 'all' ? '' : `?folderId=${folderId}`}`
  
  const response = await fetch(api);
  const { data } = await response.json();
  
  return data;
```

Reply:
삼항연산자로 처리하는게 더 깔끔해 보입니다. 참고하겠습니다!